pattern-library/html/boilerplate-header+footer+content+nav.html

```diff
@@ -0,0 +1,113 @@
+
+        <base href="{base path}">
+
+        <!-- place favicon.ico in the root directory -->
+        <link rel="shortcut icon" href="{favicon path}">
+        <!-- place touch icon in the root directory -->
+        <link rel="apple-touch-icon" href="{icon path}">
+
+        <!-- view/page metadata -->
+        <title>{page/view's title}</title>
+        <meta name="description" content="">
+
+        <!-- place primary assets (needed for or to support base view/page rendering) here -->
+        <link rel="stylesheet" href="{main css path}">
+        <script src="{main js path}"></script>
+    </head>
+
+    <body class="{app/site-section-name} {view-name} {stateful-class}">
+        <!-- CASE: if needed, provide a message for older versions of IE -->
+        <!--[if lt IE 9]>
+            <p class="alert alert-browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+        <![endif]-->
+
+        <div class="nav-skip sr-only">
+            <a class="action-skip" href="#nav-app">Skip to the main navigation</a>
+            <a class="action-skip" href="#content-main">Skip to the main content</a>
+            <a class="action-skip" href="#content-aside">Skip to the aside content</a>
+        </div>
+
+        <!-- wrapper for all UI rendered within the page/view -->
+        <div class="wrapper-view">
+
+            <!-- NOTE: wrapper elements are only needed when they need to manage the grid or larger view/page layout aspects (e.g. spanning the whole viewport) -->
+            <div class="wrapper-header-app">
+                <header class="header-app" role="banner">
+                    <!-- example: app logo/title -->
+                    <h1 class="header-app-title">
+                        <a class="logo" href="{app or company home url}">
+                            <img class="logo-img" src="{path/to/logo-img.svg}" alt="{app/site name}" />
+                            <!-- example: alternate app/site name (if not a graphical logo) -->
+                            <!-- {app/site name} -->
+                        </a>
+                    </h1>
+                    <!-- insert header content here -->
+
+                    <div class="wrapper-navigation">
+                        <nav class="nav-account" id="nav-account">
+                            <!-- NOTE: simplified user account nav example. This would be more complex in reality (with dropdowns, etc.) -->
+                            <a href="{url}">
+                                {username}
+                                <span class="sr-only">'s settings/profile</span>
+                            </a>
+                        </nav>
+                    </div>
+                </header>
+            </div>
+
+            <div class="wrapper-navigation">
+                <nav class="nav-app" id="nav-app">
```

Comment:
this must have an aria-label that describes which navigation section this is (most pages contain multiple nav elements).  An acceptable value here would be ```aria-label="Course"``` since this navigation is for the user menu.  It would be read to the user as "Course navigation".